src/envs/behavior_options.py

```diff
@@ -707,8 +707,15 @@ def graspObjectOptionModel(_state: State, env: "BehaviorEnv") -> None:
         else:
             grasp_obj_body_id = obj_to_grasp.body_id
         # 3.1 Call code that does assisted grasping
+        # bypass_force_check is basically a hack we should
+        # turn it off for the final system and use a real grasp
```

Comment:
so what's the plan with this going forward? if we are not planning to revisit, maybe we shouldn't say that we are going to turn it off for the final system?

Reply:
We will eventually fix this

Reply:
Next year